plugin/aws/ec2Instance.go

```diff
@@ -225,7 +222,24 @@ func (inst *ec2Instance) Exec(ctx context.Context, cmd string, args []string, op
 			identityfile = (filepath.Join(homedir, ".ssh", (keyname.(string) + ".pem")))
 		}
 	}
-	// Use the default user for Amazon AMIs. See above for ideas on making this more general. Can be
-	// overridden in ~/.ssh/config.
-	return transport.ExecSSH(ctx, transport.Identity{Host: hostname, FallbackUser: "ec2-user", IdentityFile: identityfile}, append([]string{cmd}, args...), opts)
+	var fallbackuser string
+	// Scan console output for user name instance was provisioned with. Set to ec2-user if not found
+	re := regexp.MustCompile(`\WAuthorized keys from .home.*authorized_keys for user ([^+]*)+`)
+	output, err := (inst.cachedConsoleOutput(ctx))
+	if err != nil {
+		activity.Record(ctx, "Cannot get cached console output", err)
```

Comment:
Travis CI is highlighting that these calls to `Record` both need to include formatting characters. So add `: %v` to the end of the string.

Reply:
Fixed